ddtrace/tracer/span.go

```diff
@@ -319,9 +321,65 @@ func (s *span) finish(finishTime int64) {
 		// not sampled by local sampler
 		return
 	}
+	if t, ok := internal.GetGlobalTracer().(*tracer); ok {
```

Comment:
I think stats should be compute before the s.context.drop. If I'm not mistaken this is the old client sampler, applying a flat sampling rate in the tracer.
We upscaled in the agent stats for these spans, it can now be counted.